packages/react-components/react-textarea/package.json

```diff
@@ -29,6 +29,7 @@
     "@fluentui/eslint-plugin": "*",
     "@fluentui/react-conformance": "*",
     "@fluentui/react-conformance-griffel": "9.0.0-beta.5",
+    "@fluentui/react-text": "9.0.0-rc.8",
```

Comment:
I think you don't need to add dependencies to package.json if you're only using them within stories.